### ISSUE TITLE
Hub to Waste Expert

### ISSUE THREAD

**Issue by nick69g** [CIWM/CIWM-Hub#11]
There is currently an issue where a users details when purchasing WE Learn are not being "pushed" from the CIWM Hub store into the Waste Expert platform.

**Comment by nick69g** [comment]
This has been resolved as we are pretty certain that the issue was due to some element of data duplication and  that it's not a domain/configuration issue. We can't really do anything else as we can't change the source code, it's just something to be monitored for the time being. Vivid have been contacted (again) regarding the source code.

**Comment by DanielHowells** [comment]
Nick, If a user purchases Waste Expert via the hub - can they now get access to Waste Expert? If they can't, they need to, so I'm not sure if we should be closing this.

**Comment by nick69g** [comment]
Yes it has been tested to work OK. When a learner purchases Waste Expert Learn via the Hub, they can then log in with their email and password as set up in the hub.

See the response I got back from the web developer:

Nick: Are you able to do another test with a unique account so we can check that the data is passing from CIWM to Waste Expert? We’ve not yet done a test that has proved successful yet have we?
Matt Tomlinson: Hi, yes, that's what I did yesterday. All worked fine.
